packages/cubejs-clickhouse-driver/driver/ClickHouseDriver.js

```diff
@@ -17,12 +17,14 @@ class ClickHouseDriver extends BaseDriver {
       ...config
     };
     this.pool = genericPool.createPool({
-      create: () => new ClickHouse(Object.assign({}, this.config, {
+      create: () => new ClickHouse({
+        ...this.config,
         queryOptions: {
           join_use_nulls: 1,
-          session_id: uuid()
+          session_id: uuid(),
+          ...this.config.queryOptions,
```

Comment:
This would allow to override `join_use_nulls` and `session_id`. Should it be moved above `join_use_nulls` instead to prevent this?

Reply:
I had thought about that, however I coded it with the view that it allows someone who knows what they are doing to to override this intentionally.